features/eolearn/features/hog.py

```diff
@@ -27,32 +30,26 @@ class HOGTask(EOTask):
 
     def __init__(
         self,
-        feature,
-        orientations=9,
-        pixels_per_cell=(8, 8),
-        cells_per_block=(3, 3),
-        visualize=True,
-        hog_feature_vector=False,
-        block_norm="L2-Hys",
-        visualize_feature_name="",
+        feature: SingleFeatureSpec,
+        orientations: int = 9,
+        pixels_per_cell: Tuple[int, int] = (8, 8),
+        cells_per_block: Tuple[int, int] = (3, 3),
+        visualize: bool = True,
+        hog_feature_vector: bool = False,
+        block_norm: str = "L2-Hys",
+        visualize_feature_name: Optional[str] = None,
     ):
         """
         :param feature: A feature that will be used and a new feature name where data will be saved. If new name is not
             specified it will be saved with name '<feature_name>_HOG'
 
             Example: `(FeatureType.DATA, 'bands')` or `(FeatureType.DATA, 'bands', 'hog')`
-        :type feature: (FeatureType, str) or (FeatureType, str, str)
         :param orientations: Number of direction to use for the oriented gradient
-        :type orientations: int
         :param pixels_per_cell: Number of pixels in a cell
-        :type pixels_per_cell: (int, int)
         :param cells_per_block: Number of cells in a block
-        :type cells_per_block: (int, int)
         :param visualize: Produce a visualization for the HOG in an image
-        :type visualize: bool
         :param visualize_feature_name: Name of the visualization feature to be added to the eopatch (if empty and
```

Comment:
i think '(' is redundant or put ')' at the end of line 52

Reply:
done